flymake-languagetool.el

```diff
@@ -52,6 +52,14 @@
   :type 'list
   :group 'flymake-languagetool)
 
+(defcustom flymake-languagetool-ignore-faces-alist
+  '((org-mode org-code org-block))
+  "Filters out errors if they are of fortified with faces in this alist.
+It is an alist of (major-mode . faces-to-ignore)"
+  :type '(alist :key-type symbol
+				:value-type (repeat symbol))
+  :group 'flymake-languagetool)
```

Comment:
1. Can you fix the tab/space issue here?
2. I think alist is clearer this way `'((org-mode . (org-code org-block)))`, WDYT?

Reply:
Sure, will get them done.